vm-setup/roles/v1aX_integration_test/tasks/verify.yml

```diff
@@ -58,13 +58,19 @@
       block: "{{ kubeconfig_secret.resources[0].data.value | b64decode }}"
 
   # Install Calico
-  - name: Download Calico manifest
+  - name: Download Calico v3.21.x manifests
     get_url:
-      url: https://docs.projectcalico.org/manifests/calico.yaml
+      url: https://docs.projectcalico.org/archive/v3.21/manifests/calico.yaml
```

Comment:
I think it would be nice if this Calico version were an Ansible 'defaults' or 'var'. Then we could also point Renovate at it.

Reply:
Agree on adding default value as ansible var.